### PR TITLE
T194 - Extend ManageWiki functionality

### PR DESCRIPTION
**Pull request by penguinstyles**
Allow ManageWiki to manage site logo

### CODE REVIEW

File: SpecialManageWiki.php [hunk 1]
```diff
@@ -109,6 +109,14 @@ function showWikiForm( $wiki ) {
 				'options' => $options,
 				'name' => 'mwLanguage',
 			),
+			'logo' => array(
+				'label-message' => 'managewiki-label-logo-url',
+				'type' => 'text',
+				'size' => 20,
+				'default' => 'https://static.miraheze.org/metawiki/3/35/Miraheze_Logo.svg',
```
Comment (Reception123) [inline SpecialManageWiki.php]: The problem with this is we'd want people to only be able to select a logo if it's uploaded under static.miraheze.org, so people can't set external links as logos. I think there should be a way to force this. If not make users write File:Logo and then convert that to the actual link.

Comment (penguinstyles) [inline SpecialManageWiki.php]: This is definitely possible. We should be able to use a validation callback to ensure that only static.miraheze.org files are set as the logo. 

See here: https://www.mediawiki.org/wiki/HTMLForm/tutorial2#Adding_Validation

**Comment by Southparkfan** [comment]
Anything that goes into the database should have the '|' escaped.

**Comment by Reception123** [comment]
This has been opened for more than 5 much, which must be a record for any PR ever made on the ManageWiki repo

It has never been merged because currently we are looking to store settings using CDB files, for several reasons. 

I was thinking about merging this, and implementing for a temporary basis, until the CDB file issue is sorted out, only it would require a lot of work to change later, which is not desired. 

Therefore, with regret, I am now closing this PR, as we are still waiting for the "CDB" method of doing this. Nice try though!

**Pull request closed without merging**